sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/disbursement-receipts-integration.scheduler.e2e-spec.ts

```diff
@@ -405,6 +466,26 @@ describe(
       expect(bdReceiptAwards).toStrictEqual({
         BCSG: 600,
       });
+      // Notification record.
+      const notification = await db.notification.findOne({
+        select: {
+          messagePayload: true,
+          createdAt: true,
+        },
+        where: {
+          createdAt: MoreThan(currentTestTime),
```

Comment:
The E2E tests usually set all the `dateSent` to some dummy date before executing and later use the `where` condition looking for the only `dateSent` as NULL.
https://github.com/bcgov/SIMS/blob/23accef93ff32c11219427dc40a3fa147f7f598d/sources/packages/backend/apps/api/src/route-controllers/student-account-applications/_test_/e2e/student-account-application.aest.controller.approveStudentAccountApplication.e2e-spec.ts#L282-L297

Reply:
The database has multiple notification records created with `dateSent` set to be `null` and notification message id with `NotificationMessageType.MinistryNotificationProvincialDailyDisbursementReceipt`, so it will not return the one record created for the test case with `findOne` method. I'm creating a variable `currentTestTime` prior to running `processor.processDisbursementReceipts`, so I am able to find the record created specifically for the test case with `createdAt: MoreThan(currentTestTime)`. We could also get the one created last with descending id order and limit 1. Please let me know the appropriate approach and I'll make the correction.
![image](https://github.com/user-attachments/assets/0fb6b9e4-99c9-41e7-812a-420acf8fc971)

Reply:
Please check the suggestion, it has two steps.
1 - At the beginning the DB will be updating all the datSent as null and ID 28 to have a dummy value.
2 - After the first step execute the find one must return one and only one record following the same idea already in place in other places.